src/main/kotlin/apply/domain/authenticationcode/AuthenticationCode.kt

```diff
@@ -20,9 +21,17 @@ class AuthenticationCode(
     @Column(nullable = false)
     val createdDateTime: LocalDateTime = LocalDateTime.now()
 ) : BaseEntity() {
+    private val expiryDateTime: LocalDateTime
+        get() = createdDateTime + EXPIRY_MINUTE_TIME
+
     fun authenticate(code: String) {
-        check(!authenticated) { "이미 인증되었습니다." }
         require(this.code == code) { "인증 코드가 일치하지 않습니다." }
+        check(!authenticated) { "이미 인증되었습니다." }
+        check(expiryDateTime > LocalDateTime.now()) { "인증 코드가 만료되었습니다." }
```

Comment:
```suggestion
        check(LocalDateTime.now().isBefore(expiryDateTime)) { "인증 코드가 만료되었습니다." }
```

이건 정말 제 개인적인 견해인데 날짜/시간은 부등호 비교보다 `isBefore` 등의 메서드 사용하는게 보기 편하더라구요.
부등호 비교가 더 읽기 좋다고 생각하시면 그대로 두셔도 좋습니다 ㅎㅎ

Reply:
오! `.isBefore` 메소드 좋네요 👍

Reply:
앗! 저는 부등호에 한 표!

Reply:
오~ 그렇다면 전 둘다 괜찮으므로 앞으로 계속 같이 작업할 제이슨의 의견에 따라 변경하겠습니다~ 혹시 다른 분들도 의견있다면 남겨주세요ㅎㅎㅎ

Reply:
저는 당장은 `isBefore` 로 비교하는게 더 빠르게 이해가는데, 익숙해지면 부등호가 더 직관적일 것 같네요~
저희끼리 볼거라면 부등호로 하고 익숙해져보는 것도(?) 좋을 것 같아요 ㅋㅋ